### DISCUSSION TITLE
What's the frequency for files to be delivered?

### DISCUSSION THREAD

**Discussion by amtillman** [CMSgov/price-transparency-guide#136] [Q&A] [answered]
How Often do files need to be delivered? Is there a set date preferred?

**Comment by ghost** [comment]
@amtillman Based on the ruling the files are to be updated monthly. @tdfow Asked a similar question, Timing of Public Facing Files - is there expectations for the files to be published on given date or will there be various dates? #54 that has not been replied to.

I am not from CMS and our in-house guidance has been we must publish the first set of the MRFs on 01/01/22 and can publish on any other date of the subsequent month.

**Comment by shaselton-usds** [comment, marked as answer]
@amtillman Thanks for the patience on this question. We've answered it here: https://github.com/CMSgov/price-transparency-guide/discussions/54

@BobSyracuse is correct that after the initial publish, you can publish subsequent files on any day that you wish, but try to keep it somewhat close to 30 days.

Documentation found [here](https://github.com/CMSgov/price-transparency-guide#timing-updates-for-machine-readable-files).

**Reply by Paul-Sousa** [reply]
@shaselton-usds can you please confirm that this means the first publication must be on 7/1/22, and for subsequent publications the date is flexible as long as it’s within ~30 days?  

Want to make sure this clarification doesn’t mean the first publication could be within 30 days of 7/1/22.

**Reply by shaselton-usds** [reply]
7/1/22 is the first day in which files need to be published and thereafter the “monthly” refresh is to refer to reasonably consistent periods of approximately 30 days, but are not specifying a particular day of the month.
